server/courselist.rb

```diff
@@ -92,6 +92,9 @@ class CourseList < Sinatra::Base
   # name of application to use for CTools HTTP direct information
   config_hash[:ctools_http_application_name] = "CTQA-DIRECT"
```

Comment:
should this and the below canvas_esb_application_name be set inside the external configuration file instead?